CMakeLists.txt

```diff
@@ -83,6 +83,11 @@ find_package(Snappy REQUIRED)
 find_package(RocksDB REQUIRED)
 find_package(Crypto REQUIRED)
 find_package(Dl REQUIRED)
+
+if(UBUNTU)
+  find_package(AptPkg REQUIRED)
```

Comment:
I'm going to add: https://github.com/facebook/osquery/commit/d2cea32644be0ce985976a2d753033bd759e0a09#diff-0dbce64b6fd0d5b6ccdd8bca55e0c482R41

Which should do the work of `find_package(AptPkg REQUIRED)` if you just need the location of `apt-pkg` AND (referring to my earlier commit) will discover a static library before the shared. :+1: